docs/source/models.rst

```diff
@@ -1247,6 +1247,61 @@ models to be interpreted as generic Python functions for inference via
 You can also use the :py:func:`mlflow.prophet.load_model()`
 method to load MLflow Models with the ``prophet`` model flavor in native prophet format.
 
+Prophet pyfunc usage
+~~~~~~~~~~~~~~~~~~~~
+
+This example uses the `example_wp_log_peyton_manning.csv <https://github.com/facebook/prophet/blob/main/examples/example_wp_log_peyton_manning.csv>` _
```

Comment:
instead of displaying the name of the dataset, perhaps use the hyperlink key to explain what the dataset is.

Reply:
Done.